File: src/main/java/ar/edu/um/tesoreria/rest/extern/model/LegajoFacultad.java
/**
 * 
 */
package ar.edu.um.tesoreria.rest.extern.model;

import java.io.Serializable;
import java.math.BigDecimal;
import java.time.OffsetDateTime;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

/**
 * @author daniel
 *
 */
@Data
@NoArgsConstructor
@AllArgsConstructor
public class LegajoFacultad implements Serializable {
	/**
	 * 
	 */
	private static final long serialVersionUID = 4563166998420069077L;

	private BigDecimal personaId = null;
	private Integer documentoId = null;
	private Integer facultadId = null;
	private Long legajoId = null;
	private Integer geograficaId = null;
	private Long numeroLegajo = null;
	private Integer lectivoId = null;
	private OffsetDateTime fecha = null;
	private Integer planId = null;
	private Integer carreraId = null;
	private String contrasenha = "";
	private Byte intercambio = 0;
	
}

File: src/main/java/ar/edu/um/tesoreria/rest/extern/model/kotlin/LegajoFacultad.kt
package ar.edu.um.tesoreria.rest.extern.model.kotlin

import java.math.BigDecimal
import java.time.OffsetDateTime

data class LegajoFacultad(

    var personaId: BigDecimal? = null,
    var documentoId: Int? = null,
    var facultadId: Int? = null,
    var legajoId: Long? = null,
    var geograficaId: Int? = null,
    var numeroLegajo: Long? = null,
    var lectivoId: Int? = null,
    var fecha: OffsetDateTime? = null,
    var planId: Int? = null,
    var carreraId: Int? = null,
    var contrasenha: String = "",
    var intercambio: Byte = 0,

)
